### PR TITLE
[script.nhlscores@matrix] 2022.4.20+matrix.1

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Add-on details:

- **General**
  - Add-on name: NHL Scores
  - Add-on ID: script.nhlscores
  - Version number: 2022.4.20+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/eracknaphobia/plugin.video.nhlscores
  
Live scoring and game updates via kodi notifications
        

### Description of changes:

            - Initial Release
        

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: script.nhlscores/resources/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# dummy file to init directory
```
Comment (enen92) [inline script.nhlscores/resources/__init__.py:1]: not needed in python3

File: script.nhlscores/resources/lib/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+# dummy file to init directory
```
Comment (enen92) [inline script.nhlscores/resources/lib/__init__.py:1]: not needed in py3

File: script.nhlscores/resources/settings.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (enen92) [inline script.nhlscores/resources/settings.xml:1]: can you update this to the new settings format? There's an addon in the forum to do it automatically on your behalf

**Review by enen92 — COMMENTED** [review]
Looks okay to me apart from the requested changes.
It would probably be a bit more clean if it had a service as the main extension point type. A script extension point could then be used to enable/disable the service. But it's fine as is.
Nice one!

**Comment by eracknaphobia** [comment]
Ok, i've made the requested changes. I'll look into using a service to integrate this in future releases. Thanks for the review!

**Review by enen92 — APPROVED** [review]

**Pull request merged**